x/dex/handler.go

```diff
@@ -1,45 +0,0 @@
-package dex
```

Comment:
I don't really understand why `x/dex/handler.go` is being deleted, is this unrelated? a legacy file cleanup?

Reply:
Legacy file cleanup.